new-feature.txt

```diff
@@ -0,0 +1 @@
+This flie created in the feature1 branch
```

Comment:
it's reall good that you have to decided to add this line of code

Reply:
Thanks P